extensions/sql-migration/src/dialog/migrationStatus/migrationStatusDialog.ts

```diff
@@ -72,7 +72,7 @@ export class MigrationStatusDialog {
 			this._view.onClosed(e => {
 				clearInterval(this._autoRefreshHandle);
 			});
-			return view.initializeModel(form);
+			return await view.initializeModel(form);
```

Comment:
Were you seeing a bug here or something? You can just return the promise directly here as it was doing before (and no need to make it async). Making it async and awaiting here is just adding extra overhead (since the async will result in it being wrapped in a promise again) and doesn't change how this functions.